### DISCUSSION TITLE
[Feature Request] Queryable Encryption Support

### DISCUSSION THREAD

**Discussion by graphicbeacon** [mongo-dart/mongo_dart#345] [Enhancements]
Hello, firstly thanks to your hard work, I've deployed a Dart service on AWS which uses mongo_dart to communicate with the database. 

Wanted to know what the roadmap for implementing support for queryable encryption looks like? I've looked in the mongo_dart API docs but only seems to have TLS encryption support.

Here's the MongoDB docs and would be great to gather your thoughts on this
* https://www.mongodb.com/docs/v7.0/core/csfle/
* https://www.mongodb.com/docs/v7.0/core/queryable-encryption/features/

Many thanks

**Comment by giorgiofran** [comment]
It is a very interesting feature.
But it seems to be also complex to develop.
At present I have not the time for analyzing this argument, but I keep this in the wish list.
